PriceKataCalc/Program.cs

```diff
@@ -78,4 +94,14 @@ internal object calcAllDiscount()
             return this.calcDiscount() + this.calcUPCDiscount();   
         }
     }
+     class Discount
+    {
+        public bool beforeTax { set; get; }
```

Comment:
Maybe using an enum here would be more readable ?

Reply:
i considered it but thought that Boolean type would lead to the same logic

Reply:
usually we use this naming convention for booleans. "Is***"